test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -49,10 +49,9 @@ const (
 var (
 	defaultMetricsCollectorConfig = metricsCollectorConfig{
 		Metrics: []string{
-			"scheduler_scheduling_algorithm_predicate_evaluation_seconds",
-			"scheduler_scheduling_algorithm_priority_evaluation_seconds",
 			"scheduler_binding_duration_seconds",
 			"scheduler_e2e_scheduling_duration_seconds",
+			"scheduler_framework_extension_point_duration_seconds",
 			"scheduler_scheduling_algorithm_preemption_evaluation_seconds",
```

Comment:
this can be removed as well (deprecated). we can get preemption latency from PostFilter extension point latency.

Reply:
Agreed, let's do that in another PR.

Reply:
SGTM. I suggested this only because you removed the predicate/priority latency metrics (also deprecated) :)

Reply:
yes, the other two that is being removed from this PR is not able to get from scheduler.

Reply:
change is here: https://github.com/kubernetes/kubernetes/pull/96447